Documentation/General/finalReport/Description.tex

```diff
@@ -1 +1,44 @@
-\chapter{Description of the developed functionalities}
\ No newline at end of file
+\chapter{Description of the developed functionalities}
+
+\section{Introduction}
+In this section we will discuss the main functionalities of our system.
+MoodCat can be split up into three main features.
+These functionalities are the ability to play music, chatting and classifying a song.
+They will be described in the sections below.
+But before explaining that, we will give a small overview of the structure of the system.
+
+%Is this needed?
+\section{Small overview}
+When a user comes at website he selects one or more moods he is in. 
+Then a list of rooms fitting that selection of moods will show.
+When clicking on a room a user joins a room where he can listen and chat with other people in the room.
+A user can say if the song is appropriate for a song. If the song is not appropriate the user gets a small pop-up on screen to tell how the song should be rated.
+\\
+MoodCat also has a game where a user gets non-rated songs and gets to classify while listening to a maximum 30 seconds snippets of songs.
+
+
+\section{Music playing}
+MoodCat is developed as a music application and thus the ability to play music is very important.
+Although, our music player is a bit different from a normal music player.
+MoodCat is a streaming service, because of that the music is synchronized with the room the player is in.  
+So you can only mute the sound and then the music will keep playing, only with no sound.
+When the internet connection of the user gets down for a few seconds or the connection is slow, when reconnection the music starts again at the moment where the whole room is.\\
+Thus, the music playing functionality of MoodCat can be best described as a radio station.
+
+\section{Chatting}
+In MoodCat there is the functionality to chat with the other people in the room.
```

Comment:
Its not a funtionality that is just there, its one of our key points. Therefore this should be stated as:
> In Moodcat you can chat with other people in the room.